wireguard-install.sh

```diff
@@ -330,7 +330,8 @@ function revokeClient() {
 	sed -i "/^### Client ${CLIENT_NAME}\$/,/^$/d" "/etc/wireguard/${SERVER_WG_NIC}.conf"
 
 	# remove generated client file
-	rm -f "${HOME}/${SERVER_WG_NIC}-client-${CLIENT_NAME}.conf"
+	find /home/ -maxdepth 2 -name "${SERVER_WG_NIC}-client-${CLIENT_NAME}.conf" -delete
```

Comment:
Might not be in `/home`

Reply:
The script saves the profiles either in `/home` or in `/root`, these lines cover both paths.

Reply:
I'm sorry I might be missing something but how does `find /home/ -maxdepth 2` cover `/root`?

Reply:
The next line 334 covers `/root`